src/NServiceBus.AcceptanceTests/Core/Sagas/When_adding_state_to_context.cs

```diff
@@ -1,15 +1,15 @@
-namespace NServiceBus.AcceptanceTests.Sagas
+namespace NServiceBus.AcceptanceTests.Core.Sagas
```

Comment:
Why were these 3 tests moved into the Core folder? Wouldn't these still be valuable to have running in downstream tests?